src/Symfony/Component/Serializer/composer.json

```diff
@@ -21,7 +21,7 @@
     "require-dev": {
         "symfony/yaml": "~2.8|~3.0",
         "symfony/config": "~2.8|~3.0",
-        "symfony/property-access": "~2.8|~3.0",
+        "symfony/property-access": "~3.1",
```

Comment:
IIUC, this is "just" to make the test pass, right? The code change itself does not need that. So, IMO, we should keep the constraint as is and tweak the test instead.